CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 0.63.0      2024-11-21
+
+* Fix WebSocket error handling
+  - If the WebSocket mechanism hit an error at the wrong time, this could cause
+    a stuck collector state, where the collector would keep running but stop
+    processing and sending snapshots
+* Track Postgres buffer cache usage
+  - This reports statistics from [pg_buffercache](https://www.postgresql.org/docs/current/pgbuffercache.html)
+    if available
```

Comment:
It might be good to mention the `max_buffer_cache_monitoring_gb` setting here. Maybe a summarized version of the PR description: https://github.com/pganalyze/collector/pull/633